src/lib.rs

```diff
@@ -743,6 +674,146 @@ fn send_tile_rebuild_tasks_system<C: OxidizedCollider>(
     heightfields.clear();
 }
 
+fn handle_geometry_result(
```

Comment:
I don't like this method, but I needed to extract it in order for it to be recursive